code/modules/migrations/SS13_Prefs/001-create.dm

```diff
@@ -136,4 +136,17 @@
 		if(!execute(sql))
 			return FALSE
 
+	sql={"CREATE TABLE "money_highscore" (
```

Comment:
This will never get executed because the DB knows how many migrations it went under. You need to make a new migration file, tick it, and it will be executed right after the commit is pushed to live.